pastebin_connector.py

```diff
@@ -65,12 +65,8 @@ def _convert_to_client_tz(self, t_str):
     def _add_file_to_vault(self, action_result, file_name, container_id, file_data):
         success = True
 
-        if hasattr(Vault, 'get_vault_tmp_dir'):
-            temp_dir = Vault.get_vault_tmp_dir()
-        else:
-            temp_dir = "opt/phantom/vault/tmp"
-
-        temp_dir = temp_dir + '/{}'.format(uuid.uuid4())
+        vault_tmp_dir = Vault.get_vault_tmp_dir()
+        temp_dir = vault_tmp_dir + '/{}'.format(uuid.uuid4())
         os.makedirs(temp_dir)

```

Comment:
Maybe this whole thing could be done more like this https://github.com/splunk-soar-connectors/microsoftwindowsremotemanagement/pull/22/files#diff-ca6e181c530d0fa3e279b1e9396a120bdd444ea4d52c1e5043f7cfa11911249dR455? Can you check?